gpkit/nomials/variables.py

```diff
@@ -166,7 +166,7 @@ def __new__(cls, shape, *args, **descr):
             vl[i].key.arraykey = arraykey
 
         if descr.pop("newvariable", True):
-            from .. import MODELS, MODELNUMS, NAMEDVARS
+            from .. import MODELS, MODELNUMS
```

Comment:
@bqpd, please confirm this is the correct fix as opposed to `NAMEDVARS[tuple(MODELS), tuple(MODELNUMS)].append(self)`.